app.ts

```diff
@@ -0,0 +1,354 @@
+/// <reference path="./interfaces/has-format-method.ts" />
+/// <reference path="./classes/table-heading-string.ts" />
+/// <reference path="./classes/render-table-headings.ts" />
+/// <reference path="./classes/render-table-rows.ts" />
+/// <reference path="./classes/get-column-headings.ts" />
+
+// Global variable
+let timeOut: ReturnType<typeof setTimeout>;
+
+/**
+   * This event executes two funtions immediately after the page loads
+   * @function HFM.getColumnHeadings This is a function that gets the column headings from the back-end
```

Comment:
I prefer this pattern for comments: @function HFM.getColumnHeadings gets the column heading ...

Reply:
Thanks. I'll quickly change the comments to your suggestion